src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -1,5 +1,54 @@
+import environment from '../../../platform/utilities/environment';
+import appendQuery from 'append-query';
 import { transformForSubmit } from 'us-forms-system/lib/js/helpers';
 
+const searchInstitutionBaseUrl = `${environment.API_URL}/v0/gi/institutions/search`;
+
+export function fetchInstitutions({ institutionQuery, page }) {
+  // const fetchUrl = institutionQuery ? `${searchInstitutionBaseUrl}?name=${institutionQuery}` : url;
+  const fetchUrl = appendQuery(searchInstitutionBaseUrl, {
+    name: institutionQuery,
+    page
+  });
+
+  return fetch(fetchUrl, {
+    headers: {
+      'X-Key-Inflection': 'camel'
+    }
+  })
+    .then(res => res.json())
```

Comment:
Should we return `res.json()` if (res.ok), otherwise return `Promise.reject(res);`?
And provide error logging in a `catch` statement?
If so, is that where we would want to trigger `LOAD_SCHOOLS_FAILED`?